src/sidebar/store/modules/annotations.js

```diff
@@ -48,17 +48,22 @@ function findByTag(annotations, tag) {
 }
 
 /**
- * Initialize the status flags and properties of a new annotation.
+ * Set custom private fields on an annotation object about to be added to the
```

Comment:
The intent is that `initializeAnnotation` (which is called by the `ADD_ANNOTATIONS` reducer) is invoked for _all_ annotations getting added to the state's collection of annotations, while the additional logic in `createAnnotation` (below) is intended for truly new (i.e. unsaved/just-created-in-the-client) annotations only.